### PR TITLE
fix java logo in index.html

### PR DESCRIPTION
**Pull request by mwasifk**
Fixing the broken image link under the Java section by replacing with another url. (Also added a credit line for the source of the image in the HTML comments).

### CODE REVIEW

**Comment by AmanSarraf** [comment]
- Please provide some description 
- Attach the related issue id to the PRs as well
I am doing this for you, 
Fixes : #23

File: index.html [hunk 1]
```diff
@@ -356,8 +356,9 @@ <h3 class="headerissue">Golang</h3>
                     <div class="cardissue">
                       <div class="cardcontentissue">
                         <a href="https://goodfirstissue.dev/language/java" class="java-link" data-toggle="modal">
-
-                        <img src="https://assets.stickpng.com/images/58480979cef1014c0b5e4901.png" height="150px" width="150px" class="text-center imageissue"/>
+                        
+                        <!--IMAGE CREDIT DUE TO SEEKLOGO.COM-->
+                        <img src="https://seeklogo.com/images/J/java-logo-7833D1D21A-seeklogo.com.png" height="150px" width="150px" class="text-center imageissue"/>
```
Comment (AmanSarraf) [inline index.html:361]: Please try to use any high quality logo, the currently used logo is stretched out and looks pixelated.

**Review by AmanSarraf — CHANGES_REQUESTED** [review]

**Comment by mwasifk** [comment]
Sorry, I'm still pretty new to open source! I just fixed it.

File: index.html (hunk 1 above)
Comment (mwasifk) [reply index.html:361]: Ok, will do.

Comment (AmanSarraf) [reply index.html:361]: commit your changes when done

**Pull request closed without merging**